From 0ed3ec45dcd68ec0c5fffe9c6070edb9d37789f0 Mon Sep 17 00:00:00 2001
From: Ram Dayal <ram.dayal@st.com>
Date: Thu, 30 May 2024 10:57:03 +0200
Subject: [PATCH] arm64: defconfig: add PCIe ASPM config in addons fragment

Enable PCIe ASPM support by default

Change-Id: I42cbc98a90b6d08b7ba19f58f7e4bf71ae8e1333
Signed-off-by: Ram Dayal <ram.dayal@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/383956
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 01786fc79508..61370862a563 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -53,6 +53,7 @@ CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
 CONFIG_PCIE_STM32=y
 CONFIG_PCIE_STM32_EP=y
+CONFIG_PCIEASPM_POWERSAVE=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PTM=y
-- 
2.39.5

